msteams-platform/promote-app-adoption.md

```diff
@@ -297,10 +297,11 @@ When your app is being rolled out or being used, your customers might have queri
 
 Enable and train your customer-facing teams such as enablement, consulting, support, and GTM functions to answer the expected support queries. Proactively establish a support process for your app. It might involve internal enablement, socializing documentation and training videos, internally or even externally published escalation matrix, and providing support information in your communications.
 
-In your app’s help documentation, consider providing some or all the following information to provide reliable support to your customers and admins:
+In your app’s help documentation, consider providing some or all the following information for your customers and admins:
 
-* Contact method with support information, for example, email, phone, or web portal.
-* Links to help documentation about rollout for admins, features for users, and troubleshooting to supporting functions.
+* Contact method such as email, phone, community forum, or web portal for customers who want to contact you for app support.
+* Explain features for users, app rollout and configuration methods for admins, and troubleshooting to supporting functions.
+* Explain the different personas that are required to configure your app. For example, an app that works with a CRM or a bug tracking system may require an admin to provision accounts for app users or to create an account for the app to fetch your organization's information.
```

Comment:
Can we expand the abbreviation CRM. Is it Customer Relationship Management?

Reply:
Updated.